processing/src/main/java/org/apache/druid/query/expression/IPv4AddressMatchExprMacro.java

```diff
@@ -104,19 +105,21 @@ public ExprEval eval(final ObjectBinding bindings)
 
       private boolean isStringMatch(String stringValue)
       {
-        return IPv4AddressExprUtils.isValidAddress(stringValue) && subnetInfo.isInRange(stringValue);
+        IPv4Address iPv4Address = IPv4AddressExprUtils.parse(stringValue);
```

Comment:
If you intend to support IPv6 here at some point in time, you'd want to use IPAddressString here.  But since it's currently IPv4-specific at this time, you could stick with IPv4Address.

Reply:
Yeah, I think IPv6 would be a separate class on its own. As things stand now, IPv4Address works well. We can perhaps revisit this when we have the IPv6 functionality in place. :)